### PR TITLE
fix: (Platform) fix combobox resize

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
https://github.com/sap/fundamental-ngx/issues/4437
#### Please provide a brief summary of this pull request.
Fixed Combobox behaviors on resize
#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

**Review by IlyaSurmay — APPROVED** [review]

**Comment by puru-hk** [comment]
In the normal mode dropdown list are showing in RTL attched the screenshot. 

image

**Comment by puru-hk** [comment]
If the dropdown item has long text it should be truncated. 

image

**Comment by puru-hk** [comment]
Tooltip is not display if the long text truncated. 

image

File: libs/platform/src/lib/components/form/combobox/commons/base-combobox.ts [hunk 1]
```diff
@@ -734,7 +730,7 @@ export abstract class BaseCombobox extends CollectionBaseInput implements AfterV
      * Assign custom templates
      * */
     private _assignCustomTemplates(): void {
-        this.customTemplates.forEach(template => {
+        this.customTemplates.forEach((template) => {
```
Comment (mykolavorobiov) [inline libs/platform/src/lib/components/form/combobox/commons/base-combobox.ts:733]: Unnecessary brackets

**Comment by Betrozov** [comment]
> Tooltip is not display if the long text truncated.
> 
> image

fixed

**Review by olshansky — APPROVED** [review]

**Review by mykolavorobiov — APPROVED** [review]

**Comment by puru-hk** [comment]
Looks good.

**Review by puru-hk — APPROVED** [review]

**Comment by xandrio** [comment]
@Betrozov 
can we update associated links with this issue?
https://github.com/SAP/fundamental-ngx/issues/4718
as I can see changes affect on that issue also.

**Pull request merged**